chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -55,7 +55,8 @@ spec:
             - "--snapshot-commitment={{ .Values.agents.core.capacity.thin.snapshotCommitment }}"
             - "--volume-commitment-initial={{ .Values.agents.core.capacity.thin.volumeCommitmentInitial }}"
             - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"{{ if .Values.agents.core.partialRebuildWaitPeriod }}
-            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}
+            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}{{ if .Values.eventing.enabled }}
+            - "--mbus-url=nats://mayastor-nats:4222"{{ end }}
```

Comment:
there's no mbus url in core agent yet...

Reply:
oh yes, this was added to for end to end testing . Its a part of @datacore-vvarakantham  PR.